src/components/dev-hub/related-articles.js

```diff
@@ -43,7 +43,6 @@ const RelatedCards = styled('div')`
 `;
 
 const getCardParamsFromRelatedType = (relatedArticle, slugTitleMapping) => {
-    console.log(relatedArticle);
```

Comment:
This was also getting noisy in logs and related articles seem to render properly now, so figured this would be a great time to remove.